ai_guide/api_attractions/views.py

```diff
@@ -64,3 +77,23 @@ def create_attraction_obj(
                 attraction=attraction
             )
         return attraction
+
+
+class TextToVoiceConverterApiView(APIView):
+
+    def get(self, request, attraction_id):
+        attraction = get_object_or_404(Attraction, id=attraction_id)
+        try:
+            file_name = attraction.object_name
+            file = aws_polly_client.get_audio(
```

Comment:
This method return type is `str` but the variable is called `file`, so looks a bit confusing.
So prob should be smt like `file_name` or `stored_file_name` not to be confused with the prev line variable